jaypeasee/jc-comp-challenge-1
# Finer Structural Page Adjustments

# Pull Request Template

#### What’s this PR do? 
- Adds appropriate margin, padding, text size, and alignment to cards.
- Replaces placeholder content in cards with unique content for each card.
- Provides rough grid structure for the `<nav/>`.
- Adjusts media queries to make the page more responsive.

#### Where should the reviewer start?

Start with the live page and compare it to the `styles.css` page.

 #### How should this be manually tested?

1. Inspect the cards against the provided design.
2. Change the size screens to see where and how the breakpoints are affecting the structure of the card columns.
3. Inspect the grid of the `<nav/>` user the developer tools.
4. Check for spelling typos or errors.
